src/transformers/data/data_collator.py

```diff
@@ -128,7 +128,7 @@ class DataCollatorForLanguageModeling:
     mlm: bool = True
     mlm_probability: float = 0.15
 
-    def __call__(self, examples: List[Union[torch.Tensor, Dict[str, torch.Tensor]]]) -> Dict[str, torch.Tensor]:
+    def __call__(self, examples: List[InputDataClass]) -> Dict[str, torch.Tensor]:
```

Comment:
This is not super helpful as a type annotation. Could we just add `List[int]` to the previous one:
`List[Union[List[int], torch.Tensor, Dict[str, torch.Tensor]]]`